.vib/apache/goss/goss.yaml

```diff
@@ -0,0 +1,14 @@
+file:
```

Comment:
This chart's service also exposes an HTTPS port, but unfortunately, we can't test both in Cypress. A good test here would be to check that said port (`apache:443`) works 🚀 . You can use the `http` GOSS functionality for that:

https://github.com/aelsabbahy/goss/blob/master/docs/manual.md#http

Reply:
I'm not sure this is what you meant, but we are setting the containerPort to 8443 and apache is not reachable using apache:443 so I decided to use the ip value from hostAliases, which is 127.0.0.1 but anyway.

Reply:
> but we are setting the containerPort to 8443 and apache is not reachable using apache:443

Apache should be reachable under `apache:443` as well:

```
$ curl -k https://apache:443
<html><body><h1>It works!</h1></body></html>
```

The intention of this test is to check that both the LoadBalancer port for HTTPs and the container are behaving properly.

- `apache` is the name of the LoadBalancer service
- `443` is the port specified in .Values.service.ports.https

Reply:
This wasn't working for me as I was only using 1 replica. It does work with 2 so I will change the expectedWorkers setting